test/components/generators/chat/test_hugging_face_local.py

```diff
@@ -487,18 +491,36 @@ def test_messages_conversion_is_called(self, mock_convert, model_info_mock):
     @pytest.mark.slow
     @pytest.mark.flaky(reruns=3, reruns_delay=10)
     def test_live_run(self, monkeypatch):
+        """Test live run with default behavior and enable_thinking."""
         monkeypatch.delenv("HF_API_TOKEN", raising=False)  # https://github.com/deepset-ai/haystack/issues/8811
+
+        # Test 1: Default behavior (no enable_thinking parameter) - should not include thinking tags
         messages = [ChatMessage.from_user("Please create a summary about the following topic: Climate change")]
+        llm = HuggingFaceLocalChatGenerator(model="Qwen/Qwen3-0.6B", generation_kwargs={"max_new_tokens": 50})
+        result = llm.run(messages)
+
+        assert "replies" in result
+        assert isinstance(result["replies"][0], ChatMessage)
+        reply_text = result["replies"][0].text
+        assert "climate change" in reply_text.lower()
+        assert "<think>" not in reply_text
+        assert "</think>" not in reply_text
 
+        # Test 2: With enable_thinking=True - should include thinking tags
```

Comment:
Sorry, I was not clear probably.

Let's have two tests (not 3):
- `test_live_run` (no thinking)
- `test_live_run_thinking` (thinking)